metaci/testresults/robot_importer.py

```diff
@@ -294,23 +319,11 @@ def export_robot_test_results(flowtest) -> None:
         "build": {
```

Comment:
We need to add the release name here before this makes it into production, but I haven't yet figured out where we're going to get it.

Reply:
FWIW, Jason said he had an idea about how to get the release by examining the PRs. He was going to try to provide some code to me soon.

Reply:
We can maybe hardcode as SFDO 234 for now and adjust later?

Reply:
Based on the discussion yesterday, we want to include the branch name in the build metadata, and let gus-bus do the logic to determine the release from that.